master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -64,8 +88,8 @@ $ kubectl get node <master_name> -o yaml
 * This install does not configure etcd TLS
 * This install expects that one Kubernetes master node has been labeled
```

Comment:
This only applies to the etcd manifest.  I wonder if this is really useful information or if it can be dropped - I would think maybe we just drop this line.